Tests/ClientTests/CreditCard/CreditCardEditViewModelTests.swift

```diff
@@ -10,7 +10,7 @@ import Storage
 
 class CreditCardEditViewModelTests: XCTestCase {
```

Comment:
We should update the name of the Test class and file too
```suggestion
class CreditCardInputViewModelTests: XCTestCase {
```